### PR TITLE
alter code to include the new spp logger and change test JSON to incl…

### PR DESCRIPTION
**Pull request by rhodriguerrier**
…ude necessary parameters

### CODE REVIEW

File: runtime_error_capture.py [hunk 1]
```diff
@@ -21,35 +21,50 @@ def handle_error(self, e, data, **kwargs):
 
     error = fields.Nested(ErrorSchema, required=True)
     sns_topic_arn = fields.Str(required=True)
+    environment = fields.Str(required=True)
```
Comment (Piwington) [inline runtime_error_capture.py:24]: Environment above Error

File: runtime_error_capture.py [hunk 2]
```diff
@@ -21,35 +21,50 @@ def handle_error(self, e, data, **kwargs):
...
         run_id = event['RuntimeVariables']["run_id"]
 
         runtime_variables = RuntimeSchema().load(event['RuntimeVariables'])
-        logger.info("Validated parameters")
-
         # Runtime variables.
         error = runtime_variables["error"]
         sns_topic_arn = runtime_variables["sns_topic_arn"]
-        logger.info("Retrieved configuration variables")
+        environment = runtime_variables["environment"]
```
Comment (Piwington) [inline runtime_error_capture.py:42]: Environment above Error

File: tests/test_runtime_error_capture.py [hunk 3]
```diff
@@ -11,6 +11,8 @@
     "RuntimeVariables": {
         "run_id": "bob",
         "sns_topic_arn": "topic_arn",
+        "environment": "sandbox",
+        "survey": "BMI_SG",
         "error": {
```
Comment (Piwington) [inline tests/test_runtime_error_capture.py:16]: Not your fault but could you fix variable order here. Should be:
Environment
Error
Run_ID
SNS_Topic_ARN
Survey

**Review by Piwington — CHANGES_REQUESTED** [review]
Code looks good just Alphabetical requests.

**Review by glanvl — APPROVED** [review]

**Review by Piwington — APPROVED** [review]

**Pull request merged**